docs/user-guide/compute-resources.md

```diff
@@ -91,10 +91,16 @@ runner (Docker or rkt).
 
 When using Docker:
 
-- The `spec.container[].resources.limits.cpu` is multiplied by 1024, converted to an integer, and
-  used as the value of the [`--cpu-shares`](
+- The `spec.container[].resources.requests.cpu` is converted to its millicore value,
+  multipled by 1024, and then divided by 1000, and used as the value of the [`--cpu-shares`](
   https://docs.docker.com/reference/run/#runtime-constraints-on-resources) flag to the `docker run`
```

Comment:
Fewer people understand cgroups?

I think it's more important to get the currently incorrect doc fixed first.

On Thursday, May 5, 2016, Vish Kannan <notifications@github.com> wrote:

> In docs/user-guide/compute-resources.md
> <https://github.com/kubernetes/kubernetes.github.io/pull/472#discussion_r62232037>
> :
>
> > @@ -91,10 +91,16 @@ runner (Docker or rkt).
> >
> >  When using Docker:
> >
> > -- The `spec.container[].resources.limits.cpu` is multiplied by 1024, converted to an integer, and
> > -  used as the value of the [`--cpu-shares`](
> > +- The `spec.container[].resources.requests.cpu` is converted to its millicore value,
> > +  multipled by 1024, and then divided by 1000, and used as the value of the [`--cpu-shares`](
> >    https://docs.docker.com/reference/run/#runtime-constraints-on-resources) flag to the `docker run`
>
> I wonder if it makes sense to have the documentation in terms of docker's
> APIs given that we have additional runtimes in the pipeline.
> Instead, why not say specify using cgroups APIs?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes.github.io/pull/472/files/7c9944da31dd1f209c5091640929db8d2da5489e#r62232037>
>